tests/server.rs

```diff
@@ -2192,31 +2190,6 @@ async fn max_buf_size() {
         .expect_err("should TooLarge error");
 }
 
-#[cfg(feature = "stream")]
-#[test]
-fn streaming_body() {
```

Comment:
I can't decide if this was meant to test `Body::wrap_stream` specifically, or if it was meant to test that the HTTP/1 server dispatcher properly handles a "streaming" (size-not-known) body. If we have other tests that effectively check that, then this one can go. If not, perhaps it could be changed to use `StreamBody`?

Reply:
I wrote a TODO to consider fixing this, as I was unable to fix it.